lib/diagnostic.js

```diff
@@ -14,9 +14,12 @@
 'use strict';
 
 const fs = require('fs'); // jshint ignore: line
-
-const sumLines = (filename, callback) => {
-  /* your response here */
+Nope;  Javascript.  Getting better at it, but not great at it.  So.  
+// const sumLines = (filename, callback) => {
+//   let newarr = process.argv.slice(2);
```

Comment:
`filename` is the thing being passed in, so we are not reading the filename from the command